### ISSUE TITLE
Advice Needed for Improving Naturalness

### ISSUE THREAD

**Issue by chankl3579** [CorentinJ/Real-Time-Voice-Cloning#709]
Hello, I am studying this repo as a part of my research project. 
Thank you for sharing this amazing repo. 

I have read some instructions in #437.
I have collected around 0.3 hours training data (not clean, with background music) of a speaker that is not seen in pretrained model. 
The speaker is an Indian American. His voice cannot be well cloned by pretrained models. (Maybe because of his accent?)
Thus I tried the single speaker finetune. 

I uploaded some audio samples: 
[Sample_360K.zip](https://github.com/CorentinJ/Real-Time-Voice-Cloning/files/6187014/Sample_360K.zip)
I also uploaded some of the real audio clips for your reference: 
[Data.zip](https://github.com/CorentinJ/Real-Time-Voice-Cloning/files/6187015/Data.zip)

I finetuned the synthesizer upon 360k steps and at this point of time the average loss is around 0.31. 
I feel it is near a local minimum. 
I am quite satisfied with the similarity during my subjective listening. 
As an objective measure, the similarity score measured by this repo is also quite good:
[https://github.com/resemble-ai/Resemblyzer](url)
But I notice that the audio generated is a little bit unnatural. 
I tried finetune the vocoder but seems not practical as the loss did not seem to be decreasing. 

Is there any way I can improve the naturalness?
So far I can only think of starting a new training to see if it can reach better minimum. 
Also may I ask is there any suggested metric to measure speech naturalness? 
I do not have budget to conduct a MOS test.

**Comment by chankl3579** [comment]
I am trying to use the new Pytorch version repo to do the same thing. 
I would close this issue until I have the result from new repo. 
Hope the new repo could give better audio.
